2025-05-27 10:56:35,172 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:00:00,600 WARNING libqtile core.py:_xpoll():L358 Shutting down due to disconnection from X server
2025-05-27 11:00:00,601 WARNING libqtile core.py:graceful_shutdown():L905 Server disconnected, couldn't close windows gracefully.
2025-05-27 11:00:00,604 WARNING libqtile lifecycle.py:_atexit():L38 Qtile will now terminate
2025-05-27 11:01:31,136 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:02:36,856 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:02:36,876 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:02:46,512 WARNING libqtile lifecycle.py:_atexit():L38 Qtile will now terminate
2025-05-27 11:02:53,185 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:03:47,822 ERROR libqtile manager.py:spawn():L1315 couldn't find `thunar`
2025-05-27 11:03:49,779 ERROR libqtile manager.py:spawn():L1315 couldn't find `thunar`
2025-05-27 11:03:50,061 ERROR libqtile manager.py:spawn():L1315 couldn't find `thunar`
2025-05-27 11:19:18,959 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:19:18,975 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:19:33,681 WARNING libqtile lifecycle.py:_atexit():L38 Qtile will now terminate
2025-05-27 11:19:39,765 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:19:39,829 ERROR libqtile hook.py:fire():L196 Error in hook startup_once
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/libqtile/hook.py", line 194, in fire
    i(*args, **kwargs)
    ~^^^^^^^^^^^^^^^^^
  File "/home/mahd/.config/qtile/config.py", line 247, in autostart
    subprocess.call(home)
    ~~~~~~~~~~~~~~~^^^^^^
  File "/usr/lib/python3.13/subprocess.py", line 395, in call
    with Popen(*popenargs, **kwargs) as p:
         ~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/subprocess.py", line 1039, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                        pass_fds, cwd, env,
                        ^^^^^^^^^^^^^^^^^^^
    ...<5 lines>...
                        gid, gids, uid, umask,
                        ^^^^^^^^^^^^^^^^^^^^^^
                        start_new_session, process_group)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/subprocess.py", line 1854, in _execute_child
    self._posix_spawn(args, executable, env, restore_signals, close_fds,
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      p2cread, p2cwrite,
                      ^^^^^^^^^^^^^^^^^^
                      c2pread, c2pwrite,
                      ^^^^^^^^^^^^^^^^^^
                      errread, errwrite)
                      ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/subprocess.py", line 1798, in _posix_spawn
    self.pid = os.posix_spawn(executable, args, env, **kwargs)
               ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: '/home/mahd/.config/qtile/autostart.sh'
2025-05-27 11:26:52,829 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:26:52,844 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:27:33,052 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:27:33,069 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:27:39,011 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:27:39,031 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:29:22,084 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:29:22,096 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:29:34,319 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:29:34,339 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:29:35,068 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:29:35,085 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:31:32,493 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:31:32,510 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:31:54,554 WARNING libqtile lifecycle.py:_atexit():L38 Qtile will now terminate
2025-05-27 11:32:01,739 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:52:16,965 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:52:16,979 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:55:21,565 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 11:55:21,581 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.globalmenu.GlobalMenu': No module named 'dbus_fast'
2025-05-27 12:07:24,337 WARNING libqtile utils.py:add_signal_receiver():L504 dbus-fast is not installed. Unable to subscribe to signals
2025-05-27 12:07:24,338 WARNING libqtile utils.py:add_signal_receiver():L504 dbus-fast is not installed. Unable to subscribe to signals
2025-05-27 12:13:48,798 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:13:48,813 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:09,572 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:09,586 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:28,259 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:28,274 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:30,736 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:30,750 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:31,283 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:31,303 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:31,725 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:31,738 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:32,106 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:32,121 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:32,489 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:14:32,504 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:19:11,540 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:19:11,554 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_volume.PulseVolume': No module named 'pulsectl_asyncio'
2025-05-27 12:20:22,598 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_extra.PulseVolumeExtra': No module named 'pulsectl_asyncio'
2025-05-27 12:20:22,618 WARNING libqtile __init__.py:import_class():L108 Unmet dependencies for 'qtile_extras.widget.pulse_extra.PulseVolumeExtra': No module named 'pulsectl_asyncio'
